frappe/public/js/frappe/model/create_new.js

```diff
@@ -61,7 +61,7 @@ $.extend(frappe.model, {
 		if(frappe.route_options && !doc.parent) {
 			$.each(frappe.route_options, function(fieldname, value) {
 				var df = frappe.meta.has_field(doctype, fieldname);
-				if(df && in_list(['Link', 'Select'], df.fieldtype) && !df.no_copy) {
+				if(df && !df.no_copy) {
```

Comment:
what is the use case for this? We had specifically set this so that other filters (percent complete) do not get set by this.